governance/working groups/index.md

```diff
@@ -0,0 +1,70 @@
+# Working groups
+
+For the UK TRE Community, a Working Group (WG) is the main method through which community outputs are created.
+
+WGs can be set up by any member of the community, and are focused on producing specific, tangible outputs within a given time period, and are modelled off the pre-existing concept of Working Groups found in communities like [DARE UK](https://dareuk.org.uk/dare-uk-launches-dynamic-collaborative-communities-invites-proposals-for-new-groups/), the [Research Data Alliance (RDA)](https://www.rd-alliance.org/groups/creating-and-managing-rda-groups/creating-or-joining-rda-working-group.html), the [W3C](https://www.w3.org/2017/Process-20170301/#GAGeneral) and more.
+
+There are two distinct processes for those considering creating WGs to consider:
+1. The establishment of the WG
+2. The community endorsement of the WG outputs
+
+As the UK TRE community's primary focus is to act as a signposting and convening body for the TRE space in the UK and beyond, we have made a conscious effort to separate out the questions of:
+- Work that is happening within the community (WG establishment)
+- Outputs/resources for which there is community consensus and endorsement (community endorsement)
+
+Therefore it is important to note that establishing a WG with the UK TRE community **does not** imply any outputs of the WG are endorsed by the UK TRE Community.
+
+
+## Process
+### Establishing a Working group
+
+1. Member(s) suggesting a working group fill in the [Working Group Charter](working-group-charter.md).
+2. The Charter is reviewed by the [Steering Group]() to ensure it:
+    - Aligns with [community principles]()
+    - Represents new work being undertaken in the community
+3. If the Charter is rejected by the [Steering Group](), feedback is provided to the submitting member(s) on why, and the Working Group returns to step 1.
+4. If the Charter is approved by the [Steering Group](), the Charter is made available for community review via an Issue on the [UK TRE GitHub](https://github.com/uk-tre/community-management) for a period of 2 weeks.
```

Comment:
There is a difference between making available and calling for feedback.

> It was on display in the bottom of a locked filing cabinet stuck in a disused lavatory with a sign on the door saying Beware of the Leopard.

Reply:
We should explicitly include in the Community main channels: mailists, slack and GH board(?)

Reply:
Community updates during events should also be used to communicate new WGs created or proposed

Reply:
I think we should setup an announcements only email list (or newsletter list), everything else is liable to be lost amongst other information.